rllib/algorithms/crr/crr.py

```diff
@@ -61,6 +61,16 @@ def __init__(self, algo_class=None):
         self.td_error_loss_fn = "mse"
         self.categorical_distribution_temperature = 1.0
 
+        self.exploration_config = {
```

Comment:
This is offline RL, it should not need self.exploration_config's default of stochasting sampling.

Reply:
Removing it does not work. I've left a TODO.